src/main/java/org/folio/persist/SubjectSourceRepository.java

```diff
@@ -0,0 +1,15 @@
+package org.folio.persist;
+
+import static org.folio.rest.persist.PgUtil.postgresClient;
+import static org.folio.services.subjectsource.SubjectSourceService.SUBJECT_SOURCE;
+
+import io.vertx.core.Context;
+import java.util.Map;
+import org.folio.rest.jaxrs.model.SubjectSource;
+
+public class SubjectSourceRepository extends AbstractRepository<SubjectSource> {
```

Comment:
@JavokhirAbdullayev  Let's add documentation for this public class here.

Reply:
i think adding doc is redundant here, because they are simple crud operations, and similar classes do not have them